src/runtime/meter-regwall-api.js

```diff
@@ -59,6 +60,11 @@ export class MeterRegwallApi {
     this.deps_
       .callbacks()
       .triggerFlowStarted(SubscriptionFlows.SHOW_METER_REGWALL);
-    return this.dialogManager_.openView(this.activityIframeView_);
+    return this.dialogManager_.openView(this.activityIframeView_).then(() => {
+      this.deps_.eventManager().logSwgEvent(AnalyticsEvent.IMPRESSION_REGWALL);
```

Comment:
Are these two events always sent together? If so, is that redundant?

Reply:
In the future if there is something else that is a regwall it can trigger IMPRESSION_{OtherRegwallKeyname}_REGWALL and IMPRESSION_REGWALL.  If someone wants to parse on "any" regwall they use the generic and if they care specific about showcase regwall they use that one.

Reply:
That makes sense! Could you add a comment explaining this?

Reply:
Done